### PR TITLE
Create folder during upload from other app

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
See #598 
@jancborchardt @davivel

### CODE REVIEW

File: src/com/owncloud/android/ui/activity/Uploader.java [hunk 1]
```diff
@@ -59,9 +51,29 @@
...
+import com.owncloud.android.authentication.AccountAuthenticator;
+import com.owncloud.android.datamodel.FileDataStorageManager;
+import com.owncloud.android.datamodel.OCFile;
+import com.owncloud.android.files.services.FileUploader;
+import com.owncloud.android.lib.common.operations.RemoteOperation;
+import com.owncloud.android.lib.common.operations.RemoteOperationResult;
+import com.owncloud.android.lib.common.utils.Log_OC;
+import com.owncloud.android.operations.CreateFolderOperation;
+import com.owncloud.android.operations.CreateShareOperation;
+import com.owncloud.android.operations.MoveFileOperation;
```
Comment (jabarros) [inline src/com/owncloud/android/ui/activity/Uploader.java]: I think some of this imports should not be added due to no functionallity related with them has been included on this PR (the functionallity added is only related with CreateFolderOperation...)

**Comment by jabarros** [comment]
Reviewed. It is a fast and easy enhancement.
@tobiasKaminsky, a rebase on develop is needed in order to have included the last changes that there are on develop branch
cc @rperezb @davivel for taking a look and prioritizing.

**Comment by tobiasKaminsky** [comment]
@jabarros Done.

**Comment by masensio** [comment]
Hi @tobiasKaminsky,

we've updated develop branch with some changes, could you please update your code with our new develop branch?

I've merged this pull request with our develop branch in a local branch and the view with the new feature(create folder during upload folder) doesn't appear. I've debugged it and the Uploader class is never used, so upload_files_layout is never showed.

**Comment by tobiasKaminsky** [comment]
@masensio I know this is confusing. And I do not know who named the class this way. But you have to share a file from another app e.g. The gallery with owncloud to see this working. 
If it is no problem for you may I ask you to update the branch to latest develop version? 
Of course only if it works for you. 

Thank you 
Tobi

**Comment by masensio** [comment]
Hi @tobiasKaminsky,

I've checked again the feature, with the steps that you've indicated and it works :)
Sorry, I've confused the feature, this title is confusing for me. I understood that we could create a new folder in the normal upload process not in the share with owncloud process.

Please, update your branch with the latest develop version.

Thanks

**Comment by tobiasKaminsky** [comment]
@masensio I have updated it and I hope that I have done everything right.
But at least there were no changes concerning this pull request...

**Comment by masensio** [comment]
@tobiasKaminsky thanks for the update.

**Comment by purigarcia** [comment]
@tobiasKaminsky can you update the branch? Thanks in advance

**Comment by davivel** [comment]
This was merged, with some additional bug fixes, in #923 .

Thanks, @tobiasKaminsky

**Pull request closed without merging**